src/pages/workspaces/workspace/LeaveWorkspaceModal.js

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react'
+import { div, h, span } from 'react-hyperscript-helpers'
+import { ButtonPrimary, spinnerOverlay } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import Modal from 'src/components/Modal'
+import { Ajax } from 'src/libs/ajax'
+import colors from 'src/libs/colors'
+import { reportError } from 'src/libs/error'
+import Events from 'src/libs/events'
+
+
+const LeaveWorkspaceModal = ({ workspace: { workspace: { workspaceId } }, onDismiss, onSuccess }) => {
+  const [leaving, setLeaving] = useState(false)
+  const helpText = 'Leave Workspace'
+
+  const leaveWorkspace = async () => {
+    try {
+      setLeaving(true)
+      await Ajax().Workspaces.leave(workspaceId)
+      Ajax().Metrics.captureEvent(Events.workspaceShare, { workspaceId })
```

Comment:
Is this supposed to be `workspaceLeave`?

```suggestion
      Ajax().Metrics.captureEvent(Events.workspaceLeave, { workspaceId })
```

Reply:
I'm too reckless when it comes to copy/pasting code.